topics/transcriptomics/tutorials/differential-isoform-expression/slides.html

```diff
@@ -0,0 +1,257 @@
+---
+layout: tutorial_slides
+logo: "GTN"
+
+title: "Identification of non-canonical ORFs and their potential biological function"
+contributors:
+  - gallardoalba
+
+priority: 4
+
+video: no
+zenodo_link: ""
+
+
+---
+
+## Index of contents
+
+1. Introduction
+
+2. Galaxy workflow
+
+---
+
+## Introduction
+
+- What do we mean by non-canonical ORFs?
+
+- Why study non-canonical ORFs?
+
+- Why have not been yet characterized?
+
+- How identify non-canonical ORFs?
+
+---
+
+## What do we mean by non-canonical ORFs?
+
+![example_text]({% link topics/transcriptomics/images/differential_isoform/slides/cambridge_dictionary.png %})
+
+.footnote[Source: [Cambridge dictionary](https://dictionary.cambridge.org/dictionary/english/canonical)]
+
+---
+
+## What do we mean by non-canonical ORFs?
+
+![example_text]({% link topics/transcriptomics/images/differential_isoform/slides/expression_distribution.png %})
+
+.footnote[Source: Erady, Chaitanya, et al. "Pan-cancer analysis of transcripts encoding novel open-reading frames (nORFs) and their potential biological functions." NPJ Genomic Medicine 6.1 (2021): 4.]
```

Comment:
the `{% cite .. %}` feature is supported in slides, perhaps it would be possible to switch to that here?

Reply:
Great, thanks!